spec/lib/rbac/filterer_spec.rb

```diff
@@ -74,6 +74,12 @@
         expect(results).to match_array [owned_vm]
       end
 
+      it "with :extra_cols finds Service" do
+        FactoryGirl.create :service, :evm_owner => owner_user
+        results = described_class.search(:class => "Service", :extra_cols => [:owned_by_current_user]).first
+        expect(results.first.attributes["owned_by_current_user"]).to be false
```

Comment:
Ok, thanks for the clarification.  So, I guess my question was really what would happen if the evm_owner => was the other user?  But I guess, you're testing when it doesn't exist, that it's non-nil and more specifically false.